lib/finitomata.ex

```diff
@@ -451,7 +485,8 @@ defmodule Finitomata do
                       on_enter: 2,
                       on_exit: 2,
                       on_terminate: 1,
-                      on_timer: 2
+                      on_timer: 2,
```

Comment:
https://github.com/am-kantox/finitomata/pull/117/files#diff-5473a8da649fad2033f7ae81c9ad3c465972e8a31ce21bd3a22c099ab063d306R476

I think explicit mention that this will not execute after restart or it being in volatile memory, might be a good addition to avoid potential confusion.

Reply:
The whole library is volatile, unless `persistency: true` is specified _and_ the actual persistency layer is fully implemented.

But I’ll mention that in docs, thanks!